### PR TITLE
added Zs3image input

### PR DESCRIPTION
**Pull request by shahnawaz-creator**

### CODE REVIEW

File: tests/api_tests/list_buckets_test.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ func TestListBuckets(testSetup *testing.T) {
 	t := test.NewSystemTest(testSetup)
 	t.Parallel()
 	t.Run("ListBuckets should return 200 when all the parameters are correct", func(t *test.SystemTest) {
-		t.Skip("skip till zs3server issues are resolved")
+		// t.Skip("skip till zs3server issues are resolved")
```
Comment (Kishan-Dhakan) [inline tests/api_tests/list_buckets_test.go:14]: Please remove the code instead of commenting it.

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Also unskip create bucket tests

File: tests/api_tests/list_buckets_test.go (hunk 1 above)
Comment (shahnawaz-creator) [reply tests/api_tests/list_buckets_test.go]: Removed

**Comment by shahnawaz-creator** [comment]
> Also unskip create bucket tests

Removed the skipping on create bucket tests

**Review by ahmedzidan — APPROVED** [review]
LGTM

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM

**Pull request merged**